src/ray/gcs/gcs_server/gcs_actor_scheduler.h

```diff
@@ -292,11 +304,42 @@ class GcsActorScheduler : public GcsActorSchedulerInterface {
   absl::flat_hash_set<NodeID> nodes_of_releasing_unused_workers_;
   /// The cached raylet clients used to communicate with raylet.
   std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool_;
-  /// The actor schedule strategy.
-  std::shared_ptr<GcsActorScheduleStrategyInterface> actor_schedule_strategy_;
   /// The cached core worker clients which are used to communicate with leased worker.
   rpc::CoreWorkerClientPool core_worker_clients_;
 };
 
+/// RayletBasedActorScheduler randomly selects a node from the node pool for the new
+/// actor. If the worker leasing fails at the selected node, the corresponding Raylet
+/// would try to reply a spill-back node.
```

Comment:
Nit, better document what `RayletBasedActorScheduler` is here, and document the behavior of SelectNode and HandleWorkerLeaseReply on those methods.